core/Updates/5.0.0-b1.php

```diff
@@ -44,17 +45,34 @@ public function doUpdate(Updater $updater)
 
     public function getMigrations(Updater $updater)
     {
+        $migrations = $this->getUpdateArchiveIndexMigrations();
+
         if ($this->requiresUpdatedLogVisitTableIndex()) {
-            return $this->getLogVisitTableMigrations();
+            return $this->getLogVisitTableMigrations($migrations);
         }
 
         return [];
```

Comment:
Here we maybe always should return the archive index migrations. Otherwise they might not be executed if someone already applied the log_table index change for some reason.

```suggestion
        return $migrations;
```